modeling-cmds/src/each_cmd.rs

```diff
@@ -522,6 +522,7 @@ pub struct MouseClick {
 }
 
 /// Enable sketch mode on the given plane.
+/// If you want to sketch on a face, use `enable_sketch_mode` instead.
```

Comment:
lol we've gotta change this, people are going to write a blog post making fun of us and they'll be right to. We can use Serde to make the underlying JSON string tags the same but use more useful names for the Rust types.

Reply:
I have an underlying issue for this, its a bigger change in that we need to evaluate where we use what in the app, converge one for a release, then we delete the old ya know or else we will break all app users https://github.com/KittyCAD/engine/issues/1774